app/mappers/user.js

```diff
@@ -0,0 +1,10 @@
+const bcrypt = require('../services/bcrypt');
+
+module.exports.create = async function create(params) {
```

Comment:
- This should not call another service, only map data
- `module.exports.create` is the same of `exports.create`
- This could be an arrow function instead of named function